### ISSUE TITLE
SIMS Application Validation - Minimum/Maximum ages

### ISSUE THREAD

**Issue by CarlyCotton** [bcgov/SIMS#2814]
See #2700 - further analysis needed re minimum age. SIMS doesn't verify and MSFAA will reject based on age.

**Current Behaviour**
SIMS does not validate the age of the students applying even if they are outside the acceptable ages. 
Students that are younger or older than expected will generate a hard reject when submitting MSFAA numbers to federal.

**Expected Behaviour**
SIMS knows the age of students prior to sending the MSFAA file and should prevent a student form progressing through the application process if outside the acceptable age range.
Provide option for manual ministry override to allow student to apply outside of supported age ranges.

**Acceptance Criteria**

**Comment by HRAGANBC** [comment]
@CarlyCotton If this is related to the minimum age requirements between 16 and 99, I would suggest we don't implement this at the MSFAA stage, but instead as an eligibility criteria/validation upon application submission. 

I.e. a student who will not be within 16 and 99 for at least one day of their study period is unable to submit an application. Although stupid, the age requirement is outlined in policy. 

In any case, enhancement? Super rare / possibly never it'll happen in first year or two.

**Comment by CarlyCotton** [comment]
@HRAGANBC Agreed - we should handle this to prevent it from getting to MSFAA/eCert. I'll re-word/tag to indicate it's a possible enhancement at the app stage .

**Comment by CarlyCotton** [comment]
Bumped to 3.0 due to expected volume of students.

**Comment by Joshua-Lakusta** [comment]
Schools should be managing this and not allowing students in. No real risk as the student gets blocked by msfaa error but should be resolved in the future soon after mvp release.
